plugins/salesforce/komand_salesforce/tasks/monitor_users/task.py

```diff
@@ -133,13 +140,36 @@ def run(self, params={}, state={}):  # noqa: C901
                     if user_login_next_page_id:
                         state[self.USER_LOGIN_NEXT_PAGE_ID] = user_login_next_page_id
                         has_more_pages = True
+
+                    self.logger.info(f"{len(response.get('records'))} users login added to output")
                     records.extend(self.add_data_type_field(response.get("records", []), "User Login"))
+
+                if removing_duplication is True:
+                    records = self.remove_duplications(records)
+
+                records = [record.__dict__ for record in records]
+
                 return convert_to_camel_case(clean(records)), state, has_more_pages, 200, None
             except ApiException as error:
                 return [], state, False, error.status_code, error
         except Exception as error:
             return [], state, False, 500, PluginException(preset=PluginException.Preset.UNKNOWN, data=error)
 
+    def remove_duplications(self, records: list) -> list:
+        """
+         Remove duplicate entries from the provided list of records.
+
+        Args:
+            records (list): A list containing the records to be de-duplicated.
+
+        Returns:
+            list: A list containing only the unique records from the input list.
+        """
+        unique_records = list(dict.fromkeys(records))
+        if len(records) != len(unique_records):
+            self.logger.info(f"Removed {len(records) - len(unique_records)} duplicate records.")
```

Comment:
Maybe extend this log line to state Removed X duplicates from a total of Y records